src/app/terms/page.tsx

```diff
@@ -0,0 +1,164 @@
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
+import { Separator } from "@/components/ui/separator";
+
+export default function TermsOfServicePage() {
+  return (
+    <div className="max-w-4xl mx-auto p-6">
+      <h1 className="text-3xl font-extrabold mb-4">利用規約</h1>
+      <div className="h-[calc(100vh-4rem)] overflow-y-auto">
+        <Card className="space-y-4 p-6">
+          {/* 第1条 */}
+          <CardHeader>
+            <CardTitle className="text-xl">第1条（適用）</CardTitle>
+          </CardHeader>
+          <CardContent>
+            <p>
+              本規約は、ユーザーと当プロジェクトとの間の本サービスの利用に関わる一切の関係に適用されるものとします。
+            </p>
+            <p>
+              当プロジェクトは本サービスに関し、本規約のほか、ご利用にあたってのルール等、各種の定め（以下、「個別規定」といいます。）をすることがあります。これら個別規定はその名称のいかんに関わらず、本規約の一部を構成します。
```

Comment:
「いかん」は「如何」だと思うのですが、漢字の方が良いかなと思います
```suggestion
              当プロジェクトは本サービスに関し、本規約のほか、ご利用にあたってのルール等、各種の定め（以下、「個別規定」といいます。）をすることがあります。これら個別規定はその名称の如何に関わらず、本規約の一部を構成します。
```